junifer/data/template_spaces.py

```diff
@@ -121,15 +124,19 @@ def get_template(
     Raises
     ------
     ValueError
-        If ``space`` is invalid.
+        If ``space`` or ``template_type`` is invalid.
     RuntimeError
-        If template in the required resolution is not found.
+        If required template is not found.
 
     """
     # Check for invalid space; early check to raise proper error
     if space not in tflow.templates():
         raise_error(f"Unknown template space: {space}")
 
+    # Check for template type
+    if template_type not in ["T1w", "whole", "gm", "wm", "csf"]:
```

Comment:
What do you mean with `"whole"`. Does it come from any toolbox? I know that usually it is named "brain" mask to reference the GM, WM and CSF masks.

Reply:
it should be "brain" then. That's the brain mask.

Reply:
Addressed.